lib/run/index.js

```diff
@@ -133,10 +133,33 @@ module.exports = function (options, callback) {
                 result[value] = true;
             }, {});
         }
+        let RequiredFolder = [];
```

Comment:
@coditva @pulkit-30 is this a pattern to have variables (not constants) start with capital case for Newman?